woodwork/tests/schema/test_schema_column.py

```diff
@@ -27,14 +27,12 @@
 
 
 def test_validate_logical_type_errors():
+    assert _validate_logical_type(None) is None
+
```

Comment:
If we only call this function with non-`None` inputs we can remove this assertion and leave in the removed block below.